### PR TITLE
Fix bugs to support feasibility cut

### PR DESCRIPTION
**Pull request by Zhanwei-Liu**
+ Replace Clp solver with Gurobi solver to retrieve unbounded extreme ray 
+ Fix bugs to support infeasible subproblem (add feasibility cuts into maser problem)
+ Add a toy example to test the feasibility cuts

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -1,10 +1,12 @@
 using Test
-
-import SimpleBenders
+include("../src/SimpleBenders.jl")
```
Comment (matbesancon) [inline test/runtests.jl]: These two lines are not needed. To run tests, check the docs on using the Julia package manager on docs.julialang.org
(I'm on my phone and can't find the proper link)

File: test/runtests.jl [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+import .SimpleBenders
 using JuMP
 
-import Clp
-import Cbc
+using Gurobi
```
Comment (matbesancon) [inline test/runtests.jl]: Can we use Highs instead of Gurobi? Overall as a rule of thumb, don't use commercial solvers in tests

**Comment by matbesancon** [comment]
As a generic way instead of Console log, we can replace this by
MOI.set(optimizer, MOI.Silent(), true)

**Comment by matbesancon** [comment]
Thanks for the contribution!

File: test/runtests.jl (hunk 1 above)
Comment (Zhanwei-Liu) [reply test/runtests.jl]: Sure. I assume you are suggesting that I should test the scripts within the project environment.  I have checked the usage of `Pkg` and removed the two lines.

File: test/runtests.jl (hunk 2 above)
Comment (Zhanwei-Liu) [reply test/runtests.jl]: You are correct! I have checked how to retrieve unbounded extreme rays and successfully replace Gurobi with HiGHS.

**Comment by Zhanwei-Liu** [comment]
> As a generic way instead of Console log, we can replace this by MOI.set(optimizer, MOI.Silent(), true)

I test the setting mentioned by you above but it seems not to work with HiGHS. I don't find a way to set the optimizer silent in the global environment. I try to create a variable called `solver` to reduce the redundant codes.

**Comment by Zhanwei-Liu** [comment]
> Thanks for the contribution!

I am just starting to learn program development and I would love to contribute my efforts to help improve this package.

File: src/master_problem.jl [hunk 3]
```diff
@@ -18,7 +18,8 @@ function benders_optimize!(m::Model, y::Vector{VariableRef}, sd::SubProblemData,
         (res, α) = optimize!(dsp, ŷ)
         if res == :OptimalityCut
             @info "Optimality cut found"
-            if η0 ≥ α' * (dsp.data.b - dsp.data.D * ŷ)
+            if η0 - α' * (dsp.data.b - dsp.data.D * ŷ) ≥ -1e-10
```
Comment (matbesancon) [inline src/master_problem.jl]:
```suggestion
            if η0 ≥ α' * (dsp.data.b - dsp.data.D * ŷ) - 1e-10
```

**Comment by matbesancon** [comment]
thanks a lot! Merged

**Pull request merged**